### DISCUSSION TITLE
Non-centered parameterization for a hierarchical regression model

### DISCUSSION THREAD

**Discussion by hyang336** [lnccbrown/HSSM#465] [Q&A] [answered]
Sorry for the potentially noob question. If I have a hierarchical regression model like v ~ 0 + x + y + (1|subID), how should I do non-centered parameterization?

**Comment by frankmj** [comment, marked as answer]
noncentered is an argument and is on by default in a hierarchical model.

Though if you want the subID terms to be hierarchical you should either do:

*  v ~ 1+ x + y + (1|subID)

that way the v_subID's will be offset from the group distribution which
will be estimated as v_Intercept. without the "1" there is no group
distribution from which the IDs will be offset.

* If you want to use the v ~ 0 +... models and still do a hierarchical
model you should actually set noncentered= false, but then use a hyperprior
for the mean of the group distribution so that it can estimate the mu from
which the subIDs are drawn.

e.g.:

in "prior": {
"1|subID": {"name": "Normal",
                              "mu": {"name": "Normal",
                                     "mu": 1,
                                     "sigma": 2},
                              "sigma": {"name": "HalfNormal",
                                        "sigma": 1,
                                        },
}

The group distributions here will be estimated as v_1|subID_mu and the
individual v's for each subject will be drawn from that centered
distribution.

* Also you might want to consider random slopes too if you want the model
to be fully hierarchical

 v ~ 1 + x + y + (1+x+y|subID)
for noncentered = true

or
 v ~ 0+ (1+x+y|subID)
for noncentered= false

(and then you would set priors also for the x and y to come from the same
hyperprior)



Michael J Frank, PhD | Edgar L. Marston Professor
Director, Carney Center for Computational Brain Science
<https://www.brown.edu/carney/ccbs>
Laboratory of Neural Computation and Cognition <https://www.lnccbrown.com/>

Brown University
website <http://ski.clps.brown.edu>
